src/type/__tests__/validation-test.js

```diff
@@ -1078,21 +1205,37 @@ describe('Type System: Field arguments must have input types', () => {
 
   it('rejects an empty field arg type', () => {
     const schema = schemaWithArgOfType(undefined);
-    expect(validateSchema(schema)).to.deep.include({
-      message:
-        'The type of BadObject.badField(badArg:) must be Input Type but got: undefined.',
-    });
+    expect(validateSchema(schema)).to.deep.equal([
+      {
+        message:
+          'The type of BadObject.badField(badArg:) must be Input Type but got: undefined.',
+      },
+    ]);
   });
 
   notInputTypes.forEach(type => {
     it(`rejects a non-input type as a field arg type: ${type}`, () => {
       const schema = schemaWithArgOfType(type);
-      expect(validateSchema(schema)).to.deep.include({
-        message: `The type of BadObject.badField(badArg:) must be Input Type but got: ${type}.`,
-      });
+      expect(validateSchema(schema)).to.deep.equal([
+        {
+          message: `The type of BadObject.badField(badArg:) must be Input Type but got: ${type}.`,
+        },
+      ]);
     });
   });
 
+  it('rejects a non-type value as a field arg type', () => {
+    const schema = schemaWithArgOfType(Number);
+    expect(validateSchema(schema)).to.deep.equal([
+      {
+        message: `The type of BadObject.badField(badArg:) must be Input Type but got: ${Number}.`,
+      },
+      {
+        message: `Expected GraphQL named type but got: ${Number}.`,
```

Comment:
Another example. This is an opportunity to improve the quality of these errors in the future